hydra-core/app/controllers/concerns/hydra/catalog.rb

```diff
@@ -1,5 +1,15 @@
 module Hydra::Catalog
   extend ActiveSupport::Concern
   include Blacklight::Catalog
-  include Hydra::Controller::SearchBuilder
+  include Blacklight::AccessControls::Catalog
+
+  # Action-specific enforcement
+  # Controller "before" filter for enforcing access controls on show actions
+  # @param [Hash] opts (optional, not currently used)
+  def enforce_show_permissions(opts={})
+    permissions = super
```

Comment:
What sort of object is `permissions`?  It looks like we're treating it as if it's a AF::Base (or SolrDocument)?  Can we name the variable to reflect that?